src/app/modules/item/http-services/get-breadcrumb.service.ts

```diff
@@ -49,6 +49,7 @@ export class GetBreadcrumbService {
               path: items.slice(0,idx).map(it => it.item_id),
               // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
               attemptId: item.attempt_id!, // the service ensures the attempt for all but last is given
+              contentType: itemRoute.contentType,
```

Comment:
It means you'll navigate to the same "content type" as the current page, right? It might not be the case as you may visit a chapter as a child of skill (maybe not for now, but it still makes sense). What's important here is to use as content type the one from the item of concern, so just infer it from the type of the item, I suppose you have it.

Reply:
GetBreadcrumb response doesn't have content type. The structure is:

attempt_id,
item_id,
language_tag,
title,

Reply:
Ok, so let's add the type... should be done in a few days.

Reply:
https://github.com/France-ioi/AlgoreaBackend/issues/773

Reply:
Done. So you now have the type in the breadcrumb service: https://france-ioi.github.io/algorea-devdoc/api/#operation/itemBreadcrumbsGet

Reply:
Fixed